test/utils.js

```diff
@@ -77,15 +79,17 @@ function extend(target) {
 	return target;
 }
 
-function loadConfig(path) {
+function loadConfig ( configFile ) {
 	try {
-		return require(path);
-	} catch (err) {
-		console.error(err.message);
-		console.error(err.stack);
-		throw new Error(
-			`Failed to load ${path}. An old test perhaps? You should probably delete the directory`
-		);
+		return require( configFile );
+	} catch ( err ) {
+		if ( err.code === 'MODULE_NOT_FOUND' ) {
+			const dir = path.dirname( configFile );
+			console.warn( `Test configuration ${configFile} not found.\nRemoving directory as this test probably no longer exists.` );
+			sander.rimrafSync( dir );
```

Comment:
I don't mean to be annoying, but I'm still just picturing cases where someone might write a test without realising they need to create a _config.js file, then watch as their carefully crafted test gets deleted... maybe we should make this even stricter and just do a straightforward `rmdir` that is allowed to fail?

Reply:
Ok, it's now as safe as it can be to still cover the specific use case. The "_actual" folder is removed via `rimraf` as this is what is not cleaned up by git, then we try to remove the test dir via `rmdir` which will fail if there are other files present.